inst/templates/ad_advs.R

```diff
@@ -0,0 +1,269 @@
+# Name: ADVS
+#
+# Label: Vital Signs Analysis dataset
+#
+# Input: adsl, vs
+
+
+# Attach/load required packages ----
+library(admiral)
+library(admiralmetabolic)
+library(tibble)
+library(dplyr)
+library(stringr)
+
+
+# Define project options/variables ----
+# Use the admiral option functionality to store subject key variables in one
+# place
+set_admiral_options(subject_keys = exprs(STUDYID, USUBJID))
+
+# Store ADSL join variables as an R object, enabling simplified usage throughout
+# the program
+adsl_vars <- exprs(TRTSDT, TRTEDT, TRT01P, TRT01A)
+
+
+# Read in data ----
+# See the "Read in Data" vignette section for more information:
+# (https://pharmaverse.github.io/admiral/articles/bds_finding.html#readdata)
+
+# Read data
+vs_metabolic <- admiralmetabolic::vs_metabolic
+adsl <- admiral::admiral_adsl
+
+# Convert SAS empty values to NA
+advs <- vs_metabolic %>%
+  convert_blanks_to_na()
+adsl <- adsl %>%
+  convert_blanks_to_na()
+
+# Merge ADSL variables (stored in `adsl_vars`) needed for ADVS
+# derivations
```

Comment:
I disagree on this, as the main reason we are adding the ADSL variables are for later derivations, and having a "subsection" about deviations could confuse. I have however changed it to "... needed for ADVS" to avoid confusion. Do you agree?

Reply:
@AndersAskeland I followed the template admiral/inst/templates/ad_advs.R, but if you disagree, I am fine with the current order